search/typesenseGoandJS/README.md

```diff
@@ -0,0 +1,14 @@
+INSTALL TYPESENSE VIA DOCKER:
+docker pull typesense/typesense:0.22.2
+
+MAKE DIRECTORY:
+mkdir /tmp/typesense-data
+
+START UP THE TYPESENSE SERVER:
+docker run -p 8108:8108 -v/tmp/typesense-data:/data typesense/typesense:0.22.2 \--data-dir /data --api-key=xyz --enable-cors
+
+index files:
+go run index.go
+
+run search:
+node search.js
```

Comment:
Cannot find module 'typesense'

Reply:
Also running "node search.js" does nothing as it's just a class definition. Add a test file like the one I put in the comments and have it run that instead.